zuul.d/tempest_multinode.yaml

```diff
@@ -1,18 +1,20 @@
 ---
 - job:
-    name: cifmw-base-multinode-tempest
+    name: cifmw-base-multinode-master-tempest
```

Comment:
Why master?

Reply:
so this is a reference to the openstack release -which is still master https://trunk.rdoproject.org/centos9-master/?C=M;O=D. master would be correct here

Reply:
but which master are we referring to though? tempest repo master? 
in our check/gate jobs we use antelope for our openstack service version.

Reply:
answer is rdo version